server/sonar-web/src/main/js/apps/quality-profiles/components/RenameProfileForm.js

```diff
@@ -49,16 +53,16 @@ export default class RenameProfileForm extends React.PureComponent {
     this.mounted = false;
   }
 
-  handleCancelClick = (event: Event) => {
+  handleCancelClick = (event /*: Event */) => {
     event.preventDefault();
     this.props.onClose();
   };
 
-  handleNameChange = (event: { currentTarget: HTMLInputElement }) => {
+  handleNameChange = (event /*: { currentTarget: HTMLInputElement } */) => {
     this.setState({ name: event.currentTarget.value });
   };
 
-  handleFormSubmit = (event: Event) => {
+  handleFormSubmit = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)